flank-scripts/src/main/kotlin/flank/scripts/ops/updatebinaries/UpdateSwift.kt

```diff
@@ -7,31 +7,30 @@ import java.nio.file.Files
 import java.nio.file.Paths
 
 private val currentPath = Paths.get("")
-private val swiftPath = Paths.get(currentPath.toString(), "swift")
+private val swiftPath =
+    if (isWindows) Paths.get(currentPath.toString(), "master-swift")
+    else Paths.get(currentPath.toString(), "swift")
 
 fun updateSwift() = if (isWindows) updateSwiftWindows() else updateSwiftOther()
 
 private fun updateSwiftWindows() {
-    println(" Will be available after #1134")
-    /*
-    TODO finish this in #1134
-    val swiftExe = Paths.get(swiftPath.toString(), "swift.exe")
-
-     if (swiftExe.toFile().exists()) {
-         println("Swift exists")
-     } else {
-         println("Downloading swift for Windows")
-         swiftPath.toFile().mkdirs()
-         downloadFile(
-             srcUrl = "https://swift.org/builds/swift-5.3-release/windows10/swift-5.3-RELEASE/swift-5.3-RELEASE-windows10.exe",
-             destinationPath = swiftExe.toString()
-         )
-     }
-
-     swiftExe.toFile().extract(swiftPath.toFile(), "zip", "gz")
-     findAndCopySwiftLicense()
-     findAndCopySwiftDemangleFile()
-     swiftPath.toFile().deleteRecursively()*/
+    val binariesPath = Paths.get(currentPath.toString(), "master.zip")
+    if (binariesPath.toFile().exists()) {
+        println("Binaries already exists")
+    } else {
+        println("Downloading binaries for windows...")
+        binariesPath.toFile().mkdirs()
+        downloadFile(
+            sourceUrl = "https://github.com/Flank/binaries/archive/master.zip",
+            destination = binariesPath.toString()
+        )
+    }
+
+    val destinationPath = Paths.get(currentPath.toString(), "master-swift")
+    destinationPath.toFile().mkdirs()
+    binariesPath.toFile().extract(destinationPath.toFile(), "zip")
+    findAndCopySwiftDemangleFile()
+    swiftPath.toFile().deleteRecursively()
```

Comment:
this function is for updating binaries in binaries repo, so we should not download here binaries from binaries repo

Reply:
are you sure it does any updating? It does not push any code to the Flank/Binary repo and simply looking through the code it downloads it copies some files to root dir and does nothing else beyond this. I looked for all instances of the #1134 that you left in code and added equivalent code for windows.

Reply:
yeah, it just downloads files and you are able to push it to Flank/binaries on your own, but if you think that it is useless, let's just delete those functions.
Otherwise they do not have any sense.

Reply:
@piotradamczyk5 they are a bit useless. Maybe we should include this functionality. We can discuss during standup.

Reply:
IMO we should delete this logic for updating files from `flank-scripts`

Reply:
will do.